treeherder/config/settings.py

```diff
@@ -77,6 +77,7 @@
     'treeherder.log_parser',
     'treeherder.etl',
     'treeherder.perf',
+    'treeherder.seta',
```

Comment:
why are we adding this back?

Reply:
The app still needs to be alive to run the migration which removes the table. The previous patch didn't contain an amendment which kept it.